### DISCUSSION TITLE
Unexpected content of the "LocalState" folder in the Ubuntu App filesystem

### DISCUSSION THREAD

**Discussion by DSBautista** [biostars/biostar-handbook-issues#233] [General]
After following the Biostar Handbook in the section "Alternative approach for accessing the Linux Files," I checked the contents of:
C:\Users\<Windows UserName>\AppData\Local\Packages\CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc\LocalState. 

The folder  ...\\LocalState\was supposed to contain subfolders **roots** and **temp**. Instead I found a Hard Disk Image file called **ext4.**

How do I correct this problem?

**Comment by ialbert** [comment]
no need to correct, use what you found

**Reply by DSBautista** [reply]
Thanks for the reply. But I'm still confused. Does the HD Image file contain the files I added to ~\myfolder while working under **bioinfo** in Linux? The image file size is under 2 gb. Will it get full eventually and cause write-failed errors while I'm processing large short-reads files?

Thanks for the help in advance.

**Reply by ialbert** [reply]
You should be able to access any other filesystem from Linux, so that alone should not be a problem. 

I am on a mac now, but I believe the other hard drives are mounted under /mnt/ or somewhere similar.

**Reply by DSBautista** [reply]
Thanks again. By the way, I can't thank you enough for your Biostar Handbook. Great work!

**Reply by ialbert** [reply]
Thanks for the kind words! Much appreciated; it takes a lot if not so visible effort to keep it all tuned and up to date.
